apps/redi-connect/src/pages/front/signup/SignUpComplete.tsx

```diff
@@ -33,15 +32,18 @@ export default function SignUpComplete() {
           <Content size="large" renderAs="div">
             <p>Your email address was successfully verified!</p>
             {userType === 'public-sign-up-mentor-pending-review' && (
-              <p>
+              <>
+                <p style={{textAlign: 'justify'}}>
```

Comment:
Do we have a good reason to add justified text alignment here and only here? If we do it here we should probably do it... _everywhere_ else in CON/TP frontends.

Reply:
Without text alignment:
<img width="371" alt="image" src="https://user-images.githubusercontent.com/51786805/201355499-a063da90-6bca-4a5e-9fed-dbbab2d9f4e3.png">

With text alignment:
<img width="373" alt="image" src="https://user-images.githubusercontent.com/51786805/201355646-131a6f4d-db8e-481d-9593-af6f2415fbde.png">

No other reason but aesthetics. Do you like the first one more?

Reply:
Oh the second one is way better. Go aesthetics!

I'd go as far as say we should do this change in all system emails. @helloanil, care to chime in?

Reply:
Yeah I agree. The second one looks way more professional. I'd also agree to change it in all emails.